tests/search_space_tests/test_group_decomposed.py

```diff
@@ -1,18 +1,17 @@
 import pytest
 
+import optuna
```

Comment:
This is not a strong opinion, but how about importing `_GroupDecomposedSearchSpace` and `_SearchSpaceGroup` directly?
```suggestion
from optuna.search_space import _GroupDecomposedSearchSpace
from optuna.search_space import _SearchSpaceGroup
```

Reply:
I fixed it. Thank you pointing out! (I changed these imports to resolve path problem in the middle of change and forgot to put back.)